Fhi.HelseId.Web/Hpr/ApprovalResponse.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+
+public class ApprovalResponse
```

Comment:
Some thoughts regarding this class; The property names does not follow the C# coding convention and I understand this is because of the json property names. I think one good approach would be to use `JsonPropertyName` which also allows us to decouple the C# class definition from the property names: https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/customize-properties?pivots=dotnet-8-0#customize-individual-property-names

Reply:
Fixed